src/libraries/System.Text.RegularExpressions/ref/System.Text.RegularExpressions.cs

```diff
@@ -243,6 +243,7 @@ public enum RegexOptions
         RightToLeft = 64,
         ECMAScript = 256,
         CultureInvariant = 512,
+        AnyNewLine = 1024,
```

Comment:
@shishirchawla Please make sure to get this new Enum documented, either with triple slash comments on top of the implementation (src file) or directly in the dotnet-api-docs repo.